src/index.css

```diff
@@ -15,6 +14,9 @@
   box-shadow: 5px 5px 10px var(--gray);
 }
 
+.info-btn{
+  background-color: var(--primary-violet) !important;
```

Comment:
Using important is considered as bad practice Vikram said in one of his sessions